fem/bilininteg.cpp

```diff
@@ -2174,20 +2178,31 @@ void VectorDiffusionIntegrator::AssembleElementMatrix(
    const int dim = el.GetDim();
    const int dof = el.GetDof();
    const int sdim = Trans.GetSpaceDim();
-   const bool square = (dim == sdim);
-   double w;
 
-   elmat.SetSize(sdim * dof);
+   // If vdim is not set, set it to the space dimension;
+   vdim = (vdim <= 0) ? sdim : vdim;
```

Comment:
What if `VQ` or `MQ` have a different` vdim`?  You should probably add some description in the hpp file